server/Html.tsx

```diff
@@ -42,6 +44,12 @@ const Html = (props: Props) => {
 		return `${manifest[`${chunkName}.${extension}`]}`;
 	};
 
+	const activeTargetSlug =
```

Comment:
This is too much logic for a single expression. I rewrote this as a helper function to get a handle on what is does:

```ts
const getActiveTargetSlug = (iniitalData: InitialData) => {
	const { scopeData, locationData } = iniitalData;
	const { activeTargetType, activePub, activeCollection, activeCommunity } = scopeData.elements;
	if (activeTargetType === 'community') {
		if (locationData.path === '/') {
			return locationData.isBasePubPub ? 'www' : activeCommunity.subdomain;
		}
		return locationData.path.split('/')[1];
	}
	if (activeTargetType === 'collection') {
		return activeCollection!.slug;
	}
	return activePub!.slug;
};
```

This more clearly breaks down into the different scope cases we need to address, but it also raises a concern for me. If `activeTargetType` is `community`, I would expect that this slug would be always be the Community's subdomain (a bit of a name kludge, but whatever) but in fact it will be a Page slug everywhere except the homepage.

More likely we want class names like `active-page-<slug>` or `active-collection-<slug>` that explicitly differentiate between these entities, and prevent quiet accidents when there's a slug collision across Pubs/Collections/Pages. It would also probably be more proper to provide HTML attributes like `data-active-page` that can be targeted with CSS, but I understand that this may be less ergonomic.

Reply:
q for you: where should this helper function go in practice? Is it okay to leave in Html.tsx for now, or should I put it in utils? I can't really see it being that useful outside of Html, but you never know.

Reply:
Definitely in `Html.tsx`...by convention we put these after the imports and type definitions but before the component itself.